inclusion_connect/users/models.py

```diff
@@ -25,8 +25,9 @@ class User(AbstractUser):
         db_index=True,
         unique=True,
     )
-    password = models.CharField("password", max_length=256)  # allow compat with old keycloak passwords
+    password = models.CharField("Mot de passe", max_length=256)  # allow compat with old keycloak passwords
     terms_accepted_at = models.DateTimeField("Date de validation des CGUs", blank=True, null=True)
+    must_reset_password = models.BooleanField("Mot de passe temporaire", default=False)
```

Comment:
Django s’occupe de `capfirst`, l’idée est que le `verbose_name` peut-être utilisé dans une phrase.
```suggestion
    password = models.CharField("mot de passe", max_length=256)  # allow compat with old keycloak passwords
    terms_accepted_at = models.DateTimeField("Date de validation des CGUs", blank=True, null=True)
    must_reset_password = models.BooleanField("mot de passe temporaire", default=False)
```